liquibase/docker-liquibase.sh

```diff
@@ -1,20 +1,8 @@
-#docker run \
-# --rm \
-# -e LIQUIBASE_URL=jdbc:postgresql://postgres:5432/gemtc \
-# -e LIQUIBASE_USERNAME=gemtc \
-# -e LIQUIBASE_PASSWORD=develop \
-# -e LIQUIBASE_CHANGELOG=liquibase-changelog.sql \
-# -v liquibase-changelog.sql:/workspace/ \
-# --link postgres:db \
-# kilna/liquibase-postgres \
-#  cat liquibase.properties
-
 docker run --rm -v $(pwd):/liquibase/ \
   --network host \
-  -e "LIQUIBASE_URL=jdbc:postgresql://postgres:5432/gemtc" \
+  -e "LIQUIBASE_URL=jdbc:postgresql://psql-test.drugis.org:5432/gemtc" \
```

Comment:
bit of a weird change given that the password doesn't work but it's a real server

Reply:
accidentally committed, changed it back